app/assets/stylesheets/styles/components/guides/_edit_guides.css.scss

```diff
@@ -1,33 +1,33 @@
 .edit-guide-app {
   margin: 2rem 0 4rem;
 
-  small.error{
+  small.error {
     margin-bottom: 0;
   }
 
-  h1{
+  h1 {
     cursor: pointer;
   }
 
-  h1.has-tip, h1.has-tip:hover{
+  h1.has-tip, h1.has-tip:hover {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line